README.md

```diff
@@ -1,26 +1,10 @@
 [![FINOS - Incubating](https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-incubating.svg)](https://finosfoundation.atlassian.net/wiki/display/FINOS/Incubating)
-[<img src="https://img.shields.io/badge/slack-@finos/compliant%20financial%20infrastructure-green.svg?logo=slack">](https://finos-lf.slack.com/messages/compliant-financial-infrastructure/)
+[<img src="https://img.shields.io/badge/slack-@finos/cloud%20service%20certification-green.svg?logo=slack">](https://finos-lf.slack.com/messages/cloud-service-certification/)
```

Comment:
Hi @J0EG - It would seem you've reverted a few changes with this latest PR. Maybe try the following to push the latest version to your downstream repo which should then reflect here. See below.

```
git pull upstream main
git push origin main
git commit -am 'merge upstream repo'
```